conda-forge.yml

```diff
@@ -1,3 +1,6 @@
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```

Comment:
Currently `linux_ppc64le` is not supported on this feedstock. Please see PR ( https://github.com/conda-forge/cuda-gdb-feedstock/pull/4 ), which attempts to add `linux_ppc64le`, but runs into issues

However it looks like adding `linux_ppc64le` to `build_platform` is resulting in builds being added for that platform. So think we need to drop it to disable `linux_ppc64le` builds

```suggestion
```